src/status_im/contacts/handlers.cljs

```diff
@@ -166,16 +174,16 @@
 (defn save-contacts! [{:keys [new-contacts]} _]
   (contacts/save-all new-contacts))
 
-(defn update-pending-status [old-contacts {:keys [whisper-identity pending] :as contact}]
-  (let [{old-pending :pending
+(defn update-pending-status [old-contacts {:keys [whisper-identity pending?] :as contact}]
+  (let [{old-pending :pending?
          :as         old-contact} (get old-contacts whisper-identity)]
     (if old-contact
-      (assoc contact :pending (and old-pending pending))
-      (assoc contact :pending pending))))
+      (assoc contact :pending? (and old-pending pending?))
+      (assoc contact :pending? (boolean pending?)))))
```

Comment:
i've wrapped `pending?` because it can be nil, but `(and old-pending pending?)` returns boolean